### ISSUE TITLE
App on Switch doesn't work

### ISSUE THREAD

**Issue by futurelucas4502** [WerWolv98/SwitchXBOXController#11]
I launch the app after installing from the hb store and cant use the dpad to change the ip and therefore click home and try to close the app im running homebrew in and i cant as the switch stops responding

**Comment by PKPenguin321** [comment]
Yep, it's been broken since I think OFW 9.0 came out

**Comment by futurelucas4502** [comment]
will it be being fixed?

**Comment by PKPenguin321** [comment]
I think it just needs to be recompiled.

**Comment by JonJaded** [comment]
Has it been recompiled?

**Comment by PKPenguin321** [comment]
No, the last release was May 1st, 2019. @WerWolv it shouldn't be too hard to recompile and push that as a new release, if you'd be inclined

**Comment by PKPenguin321** [comment]
I went ahead and recompiled it on my own since I wanted to use this again, I've tested it and it still works (but closing it will make your Switch hang on "Closing software...")
https://drive.google.com/file/d/1hueLISZO--kXpsVDWer0v_5WheEeKRaf/view
